### PR TITLE
Refactor(Tests): Structured tests to have its own location

### PR DESCRIPTION
**Pull request by bhushan**
Refactor(Tests): Structured tests to have its own location

### CODE REVIEW

File: tests/Unit/JsonApiTest.php [hunk 1]
```diff
@@ -2,46 +2,29 @@
 
 declare(strict_types=1);
 
-namespace Tests;
+namespace Tests\Unit;
```
Comment (SagarNaliyapara) [inline tests/Unit/JsonApiTest.php:5]: This looks like an API test, any reason you moved it to unit directory?

Comment (bhushan) [reply tests/Unit/JsonApiTest.php:5]: Not all tests are Functional.. its a work in progress PR 😅

Comment (bhushan) [reply tests/Unit/JsonApiTest.php:5]: but have you reviewed whole PR ? @SagarNaliyapara it will be helpful

Comment (SagarNaliyapara) [reply tests/Unit/JsonApiTest.php:5]: Yeah, I saw you've put class into separate files and few formatting changes, looks good to me

**Comment by timacdonald** [comment]
I REALLY appreciate this PR. I'm not gonna merge because I've made so many changes to things in the last week it'll probably be a nightmare to get back to normal. I promise as soon as I tag an initial version (very, very soon) I'm gonna start PR'ing all feature work that I'm doing myself so that this doesn't happen again.

That being said, I'm gonna make sure to take these changes into account and make some changes to the test setup. 

I feel like a piece of crap closing this cause I know you put work in - but know that it is really appreciated and you can consider it merged cause I'm gonna adapt what I have to your changes.

Thank you soooooo much! and sorry!!

**Comment by bhushan** [comment]
No worries 🤘🏻 @timacdonald  enjoying your series so far .. keep doing what you do !!

**Pull request closed without merging**